tests/performance/replication-test.js

```diff
@@ -9,65 +9,72 @@ module.exports = function (PouchDB, Promise) {
   var commonUtils = require('../common-utils.js');
   var log = require('debug')('pouchdb:tests:performance');
 
-  var PullRequestTestObject = function () {
-    this.localPouches = [];
-  };
+  var PullRequestTestObject = function () {};
 
-  PullRequestTestObject.prototype.setup = function (itr, gens) {
+  PullRequestTestObject.prototype.setup = function ({ itr, gens, openRevs=1, reverse=false }) {
```

Comment:
`reverse` is ambiguous... initially I thought this was referring to `descending`. What about `source="remote"` or similar?